webpack.common.js

```diff
@@ -0,0 +1,30 @@
+const { ESBuildPlugin, ESBuildMinifyPlugin } = require('esbuild-loader');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  resolve: {
+    alias: {
+      'video.js': 'video.js/core',
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'esbuild-loader',
+      }
+    ]
+  },
+  optimization: {
+    minimize: true,
+    minimizer: [new ESBuildMinifyPlugin({
+      target: 'es2015'
```

Comment:
Is this target still required with the other fixes?

Reply:
Yea, looks like it's still required.  Here's what I'm presented with for Next.js when I remove the target—

<img width="1021" alt="Screen Shot 2021-04-14 at 10 29 15 AM" src="https://user-images.githubusercontent.com/2187347/114753681-9a5ebb00-9d0c-11eb-9942-601e4320eb34.png">

To be honest, I think we want to define the target because ESBuild defaults to `esnext` (https://esbuild.github.io/api/#target).

Looking at browser support for `esnext`, the result looks spotty— https://kangax.github.io/compat-table/esnext/

For ultra compatibility, I think it would behoove use target `es2015`.